File: sach/de3304.cpp
#include <bits/stdc++.h>
#define ll unsigned long long
using namespace std;
ll a,b;
ll nen(ll n){
    for(ll i=2;i<=60;i++){
        if(n%i!=0) return i;
    }
}
ll donen(ll n){
    ll d=0;
    while (n>2) {
        n = nen(n);
        d++;
    }
    return d;
}

int main() 
{
    cin>>a>>b;
    ll s = 0;
    for (ll n=a; n<=b; n++)
        s+= donen(n);
    cout<<s;
    return 0;
}

File: sach/de3304.py
def nen(n):
    for i in range(2, 61):
        if n % i != 0:
            return i

def donen(n):
    d = 0
    while n > 2:
        n = nen(n)
        d += 1
    return d

a, b = map(int, input().split())
s = 0
for n in range(a, b + 1):
    s += donen(n)
print(s)
